contexts/PaymentContext/src/Domain/Model/BankData.php

```diff
@@ -74,6 +74,10 @@ public function hasIban(): bool {
 		return !empty( $this->getIban()->toString() );
 	}
 
+	public function hasBic(): bool {
+		return !empty( $this->getBic() );
```

Comment:
`!== ''` is clearer. `empty` is also weird:

    php > var_dump(empty('0'));
    php shell code:1:
    bool(true)